examples/docker-simple-on-ubuntu/main.tf

```diff
@@ -0,0 +1,197 @@
+variable "ubuntu_os_version_allowedvalues" {
+  type = "map"
+
+  default = {
+    "16.04.0-LTS" = "16.04.0-LTS"
+    "16.10"       = "16.10"
+  }
+}
```

Comment:
I'd argue we don't need to check this, given there's other options available (i.e. 14.04-17.04)?